routers/mapzen_results.qml

```diff
@@ -42,7 +42,7 @@ Page {
     function findRoute() {
         // Load routing results from the Python backend.
         var routePage = app.pageStack.previousPage();
-        var args = [routePage.from, gps.direction, routePage.to];
+        var args = [routePage.from, routePage.to];
```

Comment:
By dropping here `heading`, there is a case where heading would be missed:

* you have someone in the car finding a route while you drive. However, without heading info, the calculated route could be in the other direction. So, we have the same issue as with rerouting.

On the other hand, heading info on GPS could be outdated. For example, 

* we could have had a coffee stop and calculated a route to a new destination. The heading data is not relevant since you can turn around and take the best route.

By dropping heading from here as well as other `_result` pages, we select the solution that prefers the second case. In the end, its a political decision that has to be made at this stage. Just wanted to point out that through this commit, its made. I am fine either way since re-routing should make up to a `wrong` route case.